src/Hazelcast.Net/Serialization/ConstantSerializers/ConstantSerializerDefinitions.cs

```diff
@@ -43,6 +51,29 @@ public void AddSerializers(SerializationService service)
 
             service.RegisterConstantSerializer<Guid>(new GuidSerializer());
             service.RegisterConstantSerializer<KeyValuePair<object, object>>(new KeyValuePairSerializer());
+
+            //TODO: proper support for generic types
```

Comment:
What does it mean proper support?

Reply:
Interestingly, I have no idea, this TODO and the one a few lines below were copied over from the very original project (v3?) when I joined HZ. I don't understand them either, so I am going to update this PR to remove them entirely.

Reply:
Have pushed the change.